artemis/reporting/modules/bruter/translations/pl_PL/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,161 @@
+#: artemis/reporting/modules/bruter/template_exposed_archive.jinja2:2
```

Comment:
Should we upload `.po` files to the repository?

Reply:
That's what I'm scared about - ppl will change code without generating new `.po` files. When you don't have `.po` stored in repo you are forced to generate them fresh, meaning they will always be up-to-date.

Reply:
We still have to store the en->pl translation somewhere, and regardless where we store, the problem of diverging code vs translation would appear. Therefore I added a linter step for that.